frontend/src/firebase/actions.ts

```diff
@@ -368,7 +369,36 @@ export const removeSubTask = (
 
 /*
  * --------------------------------------------------------------------------------
- * Section 3: Other Compound Actions
+ * Section 3: Groups Actions
+ * --------------------------------------------------------------------------------
+ */
+
+/**
+ * Send an invitation to a user to join a group.
+ * @param groupID Document ID of the group's Firestore document. The user calling this function must
+ *                be a member of this group.
+ * @param userName The name of the user sending the invitation (in English)
+ * @param invitee The full Cornell email of the user receiving the invitation (all lowercase)
+ */
+export const sendInvite = async (
+  groupID: string, userName: string, invitee: string,
+): Promise<void> => {
+  const newInvitation: FirestorePendingGroupInvite = {
+    group: groupID,
+    inviterName: userName,
+    invitee,
+  };
+  database.pendingInvitesCollection().doc().set(newInvitation);
+};
+
+export const rejectInvite = async (inviteID: string): Promise<void> => {
+  const invitations = await database.pendingInvitesCollection().doc(inviteID);
+  invitations.delete();
```

Comment:
I think it should be:

```suggestion
  await database.pendingInvitesCollection().doc(inviteID).delete();
```

You don't need to await on a document reference.